cypress/integration/chi-js/sidenav.spec.js

```diff
@@ -0,0 +1,204 @@
+import { SIDENAV_COMPONENT_SELECTOR } from '../../../src/chi/javascript/components/sidenav';
+const drawerLiTitleSelector = ' .chi-sidenav__drawer-list li div.chi-sidenav__title';
+const drawerDropDownTitle   = '.chi-sidenav__drawer-item-list ul.chi-tabs li a';
+const drawerLiMainMenuTitleSelector = ' .chi-sidenav__drawer-list li span.chi-sidenav__title';
+
+describe('Sidenav', function() {
+  before(() => {
+    cy.visit('/tests/js/side-nav.html');
```

Comment:
```suggestion
    cy.visit('/tests/js/sidenav.html');
```
The build failed because Cypress is trying to visit side-nav.html as you had before.. Since we changed the filename, html filename is also updated. This change fixes the issue. Thnks!

Reply:
Updated. Thanks